sdk/communication/communication-phone-numbers/test/public/operators.spec.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { matrix } from "@azure/test-utils";
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { Context } from "mocha";
+import { createRecordedOcClient, createRecordedOcClientWithToken } from "./utils/recordedClient";
+import { OperatorConnectClient } from "../../src";
+
+matrix([[true, false]], async function (useAad) {
+  describe(`OperatorConnectClient - get operators list${useAad ? " [AAD]" : ""}`, function () {
```

Comment:
nit: The output here if `useAad` is false would be `get operators listcan retrieve operators list`. I think a blank space in the negative case would be necessary.

Reply:
If title of spec is concatenated in some other output, then "true" case would also need whitespace in the end (like " [AAD] ") 
When execution in console output looks like:
"""
OperatorConnectClient - consent operations                                                                                                                                             
   √ can retrieve consents list (1001ms)                                                                                                                                                
   1) can get consent                                                                                                                                                                   
   2) can create consent and get it back       
"""

Reply:
Oh I see. I thought that the `can retrieve consents list` would be printed at the same line. So this can be ignored :)